sql/updates/latest-dev.sql

```diff
@@ -27,3 +26,16 @@ BEGIN
     END IF;
 END
 $$;
+
+-- Update drop_chunks API
+DROP FUNCTION IF EXISTS @extschema@.drop_chunks;
+CREATE OR REPLACE FUNCTION @extschema@.drop_chunks(
+    relation               REGCLASS,
+    older_than             "any" = NULL,
+    newer_than             "any" = NULL,
+    verbose                BOOLEAN = FALSE,
+    created_before         "any" = NULL,
+    created_after          "any" = NULL,
+    force                   BOOLEAN = FALSE
+) RETURNS SETOF TEXT AS '@MODULE_PATHNAME@', 'ts_chunk_drop_chunks'
+LANGUAGE C VOLATILE PARALLEL UNSAFE;
```

Comment:
```suggestion
```
latest-dev doesnt need create function statements, functions will be created automatically in update scripts